### ISSUE TITLE
Cant see properly Active Devices info

### ISSUE THREAD

**Issue by datsoy** [jellyfin/jellyfin-web#3370]
### Describe the bug

Backgroud color for active devices is dark to see info.
![Screenshot_20220130-182943809 (1)](https://user-images.githubusercontent.com/67457715/151700511-614183f6-80af-44aa-b64d-1e1157e92059.jpg)
![Screenshot_20220130-205638901 (1)](https://user-images.githubusercontent.com/67457715/151700649-3d460254-1f69-46b8-8d7e-23a4e08bc36c.jpg)

### Application version

2.4.2

### Where did you install the app from?

F-Droid

### Device information

Oneplus 6T

### Android version

Android 11, Crdroid7.13

### Jellyfin server version

10.7.7 docker linuxserver

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [ ] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by nielsvanvelzen** [comment]
Confirmed when using the light theme for the dashboard

**Comment by YasarArafathAli** [comment]
hey @nielsvanvelzen, I would like to work on this.. It would be great if you'd guide me.

**Comment by nielsvanvelzen** [comment]
I can't really guide you with it, I don't work on the code for jellyfin-web. It should be somewhere in the CSS for the theme though.

**Comment by YasarArafathAli** [comment]
i would like to work on this, as this is my first contribution, it would be a good one. thanks.

**Comment by nyanmisaka** [comment]
https://github.com/jellyfin/jellyfin-web/blob/master/src/themes/light/theme.css

**Comment by Sync271** [comment]
@datsoy this issue section is for the **Web Version of Jellyfin**, I've created an issue on https://github.com/jellyfin/jellyfin-android/issues/617 and credited it to you.

**Comment by Sync271** [comment]
From the looks of it, it's already been fixed and there hasn't been a stable release after that
https://github.com/jellyfin/jellyfin-web/blob/9d3a12237af75f36b7a9985453e9a01db3a03c7c/src/assets/css/dashboard.scss#L353-L356
![image](https://user-images.githubusercontent.com/67158080/152437433-dbe46163-9f43-4f85-a9dc-f7777f34afdd.png)
